### PR TITLE
Print a user-friendly error message when URL is not found

### LINKED ISSUE #6: Bitbucket repo throws TypeError
```
TypeError: Expected a `target`
at module.exports (/home/developer/.nvm/versions/node/v9.3.0/lib/node_modules/gh-home/node_modules/open/index.js:18:9)
at __dirname (/home/developer/.nvm/versions/node/v9.3.0/lib/node_modules/gh-home/cli.js:45:9)
at <anonymous>
```

Not exactly an important issue since it's not github anyway, but maybe it should at least fail gracefully.

### PR DESCRIPTION
**Pull request by aviskarkc10**
Fixes #6

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -42,6 +42,10 @@ const repo = cli.input[0];
 			process.exit(1);
 		}
 
-		await open(url);
+		if (url) {
```
Comment (sindresorhus) [inline cli.js]: Would be more readable to do an early return.

File: cli.js [hunk 2]
```diff
@@ -42,6 +42,10 @@ const repo = cli.input[0];
...
+			await open(url);
+		} else {
+			console.error('Couldn\'t find the repo\'s url. Ensure you are inside a github repo.');
```
Comment (sindresorhus) [inline cli.js]: Add `process.exit(1)` too.

Comment (sindresorhus) [inline cli.js]:
```suggestion
			console.error('Couldn\'t find the repo\'s GitHub URL. Ensure you are inside a Git repo that points to GitHub.');
```

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: cli.js (hunk 1 above)
Comment (aviskarkc10) [reply cli.js]: fixed

File: cli.js (hunk 2 above)
Comment (aviskarkc10) [reply cli.js]: fixed

Comment (aviskarkc10) [reply cli.js]: fixed

File: cli.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const repo = cli.input[0];
 		if (repo.includes('/')) {
 			await open(`https://github.com/${repo}`);
 		} else {
-			const {stdout: user} = await execa('git', ['config', '--global', 'github.user']);
+			const { stdout: user } = await execa('git', ['config', '--global', 'github.user']);
```
Comment (sindresorhus) [inline cli.js]: Don't do unrelated changes.

File: cli.js [hunk 4]
```diff
@@ -42,6 +42,11 @@ const repo = cli.input[0];
 			process.exit(1);
 		}
 
-		await open(url);
+		if (!url) {
+			console.error('Couldn\'t find the repo\'s GITHUB URL. Ensure you are inside a github repo.');
```
Comment (sindresorhus) [inline cli.js]: This is not what I wrote in the suggestion.

File: cli.js (hunk 3 above)
Comment (aviskarkc10) [reply cli.js]: fixed

File: cli.js (hunk 4 above)
Comment (aviskarkc10) [reply cli.js]: fixed

**Comment by sindresorhus** [comment]
Linting is not passing. Always run `npm test` locally before pushing commits.

**Review by sindresorhus — APPROVED** [review]

**Pull request merged**